WorkflowExecAgent/docker_compose/docker_compose.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  supervisor:
+    image: opea/agent-langchain:latest
+    container_name: supervisor-agent-endpoint
+    volumes:
+      - ${WORKDIR}/GenAIComps/comps/agent/langchain/:/home/user/comps/agent/langchain/
+      - ${TOOLSET_PATH}:/home/user/tools/
+    ports:
+      - "9090:9090"
+    ipc: host
+    environment:
+      ip_address: ${ip_address}
+      strategy: workflow_executor
```

Comment:
My recommendation is to instantiate supervisor agent and the worker agents as separate react agents the existing react_langgraph or react_llama strategy in the agent component. Each worker agent can be registered as a tool for the supervisor agent in the hierarchical architecture that we currently support. My recommendation is not to add this new "workflow_executor" strategy. We just need to support custom prompt templates, which is already planned for v1.1 release.

Reply:
Updated strategy to be a single-tool, single react-langgraph